README.md

```diff
@@ -1865,6 +1865,9 @@ The following extractors use this feature:
 #### nhkradirulive (NHK らじる★らじる LIVE)
 * `area`: Which regional variation to extract. Valid areas are: `sapporo`, `sendai`, `tokyo`, `nagoya`, `osaka`, `hiroshima`, `matsuyama`, `fukuoka`. Defaults to `tokyo`
 
+#### nflplusreplay
+* `type`: Type(s) of game replays to extract. Valid types are: `full_game`, `full_game_spanish`, `condensed_game` and `all_22`. You can use `all` to extract all available replay types, which is the default
```

Comment:
Is this not possible with `--match-filter`. It doesn't look like the arg even reduces network calls. Perhaps it can be set to tags/categories

Reply:
The idea behind the extractor-arg is that users could add it to their config without it impacting other sites, since it is unlikely any user wants to download 4 slightly different versions of the same 4+GB game

Reply:
Here's a diff where I remove the extractor-arg and add a `media_type` field for `--match-filter`:
https://github.com/bashonly/yt-dlp/compare/fix/nfl...bashonly:yt-dlp:fix/nfl-media-type

I'd be OK with this if we must do it this way, but what I really don't like about it is that the `media_type`s for this extractor will not be documented anywhere, so most users will have no idea how to use `--match-filter` (or that they should use `--match-filter` in the first place to select replay types). With extractor-arg, at least the replay types would be documented in README

Reply:
Merged `media_type` branch into the PR and added `--match-filter` hint output

Reply:
I agree on that the extractor arg is the most user friendly alternative. The extractor arg still allows you to select each variant and it is easier to use in this context